src/Microsoft.Azure.WebJobs.Host/WebJobs.Host.csproj

```diff
@@ -367,7 +378,7 @@
     <Compile Include="Bindings\ValueBindingContext.cs" />
     <Compile Include="Bindings\IWatcher.cs" />
```

Comment:
I haven't made IWatcher functionality public yet (ServiceBus didn't use it). We can open that up as well when it is needed by other extensions. Likely there will be other instances of things we need to open up. We'll only discover these as we/others write extensions.